spec/lib/bank_accounts/account_spec.rb

```diff
@@ -0,0 +1,74 @@
+require 'spec_helper'
+
+describe BankAccounts::Account do
+
+  it "responds to #new" do
+    expect(BankAccounts::Account).to respond_to(:new)
+  end
+
+  it "responds to #new with two arguments" do
+    expect(BankAccounts::Account).to respond_to(:new).with(2).arguments
+  end
+
+  let(:bob) { BankAccounts::Account.new(1, 1000) }
+
+  it "contains @id" do
+    expect(bob.id).to eq(1)
+  end
+
+  it "contains @balance" do
+    expect(bob.balance).to eq(1000)
+  end
+
+  it "cannot initialize with a negative initial_balance" do
+    expect{BankAccounts::Account.new(2, -1000)}.to raise_error(ArgumentError)   #raise_error is the same as raise_exception
+  end   # Why does raise_error use expect{} vs expect()???
```

Comment:
The use of curly braces `{ }` as the argument to `expect` means that you're expecting whatever gets executed within those braces to do a thing (in this case, raise an error), whereas `expect` with parens `()` means that Rspec is going to compare the *return value* of whatever is inside the parens with whatever you are setting up your expectation to be.

Here's an answer on [stackoverflow](http://stackoverflow.com/questions/19960831/rspec-expect-vs-expect-with-block-whats-the-difference) that might be a bit easier to understand than the rspec docs. Let me know if that makes sense!